vignettes/function-flow.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+title: "Function Flow"
+output: rmarkdown::html_vignette
+bibliography: references.bib
+csl: vancouver.csl
+vignette: >
+  %\VignetteIndexEntry{Function Flow}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
```

Comment:
I think that an introduction to what this vignette includes would be nice here :)

Reply:
Wanna take a stab at it? 😛